.github/workflows/build.yml

```diff
@@ -36,6 +38,7 @@ jobs:
           fi
           if [ ${{ matrix.os }} == windows-latest ]; then
             echo os=windows >> "$GITHUB_ENV"
+            echo "CACHE_KEY=${RUNNER_OS}-${ImageVersion}" >> $GITHUB_ENV
```

Comment:
Out of interest, is there any harm (or performance hit) in setting CACHE_KEY on Mac even if we don't use it? My thinking is that we might one day use it and its value isn't specific to Windows, so we may as well just set it across the board. That way, we could do it in the env: block instead of doing it conditionally here.

Reply:
I don't think we can do it in the env block. The runner os can be accessed with runner.os, but I'm afraid the image version is not available in the env block. Will test, though.